apps/modernization-ui/src/apps/page-builder/pages/QuestionLibrary/QuestionLibrary.tsx

```diff
@@ -36,6 +43,7 @@ export const QuestionLibrary = ({ hideTabs, modalRef }: any) => {
                 <QuestionLibraryTable
                     summaries={summaries}
                     qtnModalRef={modalRef}
+                    onAddQuestion={onAddQuestion}
                     pages={{ currentPage, pageSize, totalElements }}
                 />
             </div>
```

Comment:
Issues(blocking): I think in the design it asks to capitalize the "you", it seems capitalized in this file but I don't think it is taking the title from this file but from the QuestionLibraryTable file.